LedApi/src/api.py

```diff
@@ -56,14 +55,8 @@ def theater_chase() -> Response:
 
 @app.route("/rainbow")
 def rainbow() -> Response:
-    led_request = LedRequest("rainbow", wait_ms = 20)
-    publish("leds", json.dumps(led_request.__dict__))
-    return Response(status=200)
-
-
-@app.route("/sleep")
-def rainbow() -> Response:
-    led_request = LedRequest("rainbow", wait_ms = 350)
+    body = request.get_json
```

Comment:
should be `get_json()`

Make sure you run this locally to test before committing changes. The interpreter would've picked up on this error